packages/devtools_app/lib/src/inspector/layout_explorer/flex/flex.dart

```diff
@@ -922,17 +996,16 @@ class FlexChildVisualizer extends StatelessWidget {
   const FlexChildVisualizer({
```

Comment:
The flex child visualizer passed too many colors in when what we really wanted was to render based on the state. This resulted in color logic duplicated 3X